av1/av1codecconfigurationrecord.go

```diff
@@ -0,0 +1,99 @@
+package av1
+
+import (
+	"io"
+
+	"github.com/Eyevinn/mp4ff/bits"
+)
+
+// DecConfRec - AV1CodecConfigurationRecord
```

Comment:
Since the name is different from AVC and HEVC, I think we we should use a name that is closer to the name in standard like "CodecConfRec", so that it becomes av1.CodecConfRec when used.